lib/workflow/runner.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+module Workflow
+  class Runner
+    QUEUE = 'tasker'
+    attr_accessor :internal_options, :options, :current_step, :current_task
+    include Sidekiq::Worker
+    sidekiq_options queue: QUEUE
```

Comment:
This is more of a devops question (cc @jkassemi) since the runner may be working for awhile, do we have separate Redis instances for cache data that is ok to push out of memory (sessions, http responses) and more permanent data like sidekiq jobs? see [sidekiq wiki doc](https://github.com/mperham/sidekiq/wiki/Using-Redis)